src/Workspaces/Core/Portable/Formatting/Formatter.cs

```diff
@@ -360,5 +361,22 @@ private static IEnumerable<TextSpan> AggregateSpans(IEnumerable<TextSpan> spans)
 
             return aggregateSpans;
         }
+
+        /// <summary>
+        /// Organizes the imports in the document.
+        /// </summary>
+        /// <param name="document">The document to organize.</param>
+        /// <param name="cancellationToken">An optional cancellation token.</param>
+        /// <returns>The document with organized imports.</returns>
```

Comment:
```suggestion
        /// <returns>The document with organized imports. If the language does not support organizing imports, or if no changes were made, this method returns <paramref name="document"/>.</returns>
```

:memo: We should confirm that the method returns `document` without changes (reference equality) for the case where no changes are made.